src/Command/Rest/EnableCommand.php

```diff
@@ -86,7 +87,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $question = new ChoiceQuestion(
```

Comment:
same as the other comment
@enzolutions: We should replace this with $out->choice method